src/mixins/itext_behavior.mixin.ts

```diff
@@ -1061,8 +992,8 @@ export abstract class ITextBehaviorMixin<
     }
     this.fire('editing:exited');
     isTextChanged && this.fire('modified');
+    this._canvasMoveDisposer && this._canvasMoveDisposer();
     if (this.canvas) {
-      this.canvas.off('mouse:move', this.mouseMoveHandler);
```

Comment:
changed this bit to use a disposer `_canvasMoveDisposer` in order to safeguard from a case where while editing text it is removed from canvas. I will add a test for it.

Reply:
added